EIPS/EIP-draft-getLogs-by-block-hash.md

```diff
@@ -0,0 +1,50 @@
+## Preamble
+```
+EIP: <to be assigned>
+Title: Add `blockHash` to JSON-RPC filter options.
+Author: Micah Zoltu
+Type: <Standard Track | Informational | Meta>
```

Comment:
This needs setting - 'standard track', presumably.

Reply:
Bleh, just noticed that this comment today.  :'(  Not sure how I missed it in my inbox.